tuist/tuist
# Fix caching tuist

### Short description 📝

This PR does two things:
- Fixes `tuist` project definition, so replacing individual binaries works correctly
- Sets up pipelines to run `cache warm` both for Sillicon and Intel machines
- `tuist test` should now reuse these binaries when available, making those tests faster

The only downside of these changes is that now we have lots of warnings that dependencies are imported by multiple targets. We should follow up on this PR and make it possible to selectively disable this warning for frameworks where you know duplicated symbols won't cause issues.

### How to test the changes locally 🧐

- CI should pass
- If you can connect to our `tuist` Tuist Cloud instance, run `tuist generate tuist` -> you should be able to run the CLI with the downloaded binaries.

### Contributor checklist ✅


### Reviewer checklist ✅
